### PR TITLE
Add features to the home-page

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
-  Made a home-page for the portfolio project 
- Add styles to the home-page
- Made a navigation bar and a description about myself
- Made social media handles 
- Set up linters for this project and made sure it works correctly
- Made a readme.md file for my project  in a descriptive manner

### CODE REVIEW

**Comment by Rishi-Mishra0704** [comment]
-  Fix Eslint linter error 
-  Made index.js to use it later in the project
-  Index.js also helped in solving the eslint issues
- Setup eslint configurations

File: index.html [hunk 1]
```diff
@@ -20,10 +20,16 @@
         <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
         <li class="wifi"><img src="assets/Wifiwifi.png"></li>
         <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would suggest you kindly remove this part from your project as it is not part of the project design and requirement, It was actually there in the Figma design to show you that the design is on a mobile view.
Kindly remove it 👍 👍

File: index.html [hunk 2]
```diff
@@ -20,10 +20,16 @@
...
+      </div><br><br>
+      <div class="nav">
+        <h1 class="my-logo">My logo</h1>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you make this an `anchor tag`. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👍 👍

File: index.html [hunk 3]
```diff
@@ -20,10 +20,16 @@
...
+        <img src="assets/Unionnavbar.png" class="navbar">
```
Comment (brainconnect93) [inline index.html:20]: - [ ] I would kindly suggest you wrap your `hamburger` icon inside a `nav` tag, The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. 👍 👍

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @BahirHakimy, 👋 

Good job so far, You really made a great effort working on your project, it looks really awesome! but there are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] Great effort working on your project. But current there is a missing aspect called `Let's Connect` in your project, kindly add it. Also the `social media icons` should be increase to 5, kindly add missing ones there and make sure it properly aligned as we have it in the Figma design.

Your Design 👇 

rishi

Figma Design 👇 



- [ ] Secondly Kindly work on the `font family`, `font size`, and `font weight`, make sure everything go with what we have in the Figma design.

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rishi-Mishra0704** [comment]
- Remove the mobile bar design such as battery , wifi and time etc
- Put the "My-logo" part in an anchor tag
- Put the navigation bar in nav tag as was required

File: index.html [hunk 4]
```diff
@@ -15,19 +15,19 @@
...
   <body>
     <header>
-      <div class="mobile-bar">
-        <li class="time"><img src="assets/Bars/Status Bars/↳ Timemobile-top.png"></li>
-        <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
-        <li class="wifi"><img src="assets/Wifiwifi.png"></li>
-        <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
-      </div>
+      <nav class="nav">
+       <a href="#"><h1 class="my-logo">My logo</h1></a>
```
Comment (binhussen) [inline index.html:19]: - [Optional] Great job for adding `nav`, I think the logo is not part of the navigation so make the `a` logo outside the `nav` tag. there will be additional navigation which you will add later 👍👍

File: style.css [hunk 5]
```diff
@@ -3,122 +3,198 @@
...
+  justify-content: space-between;
+}
+
+.my-logo {
+  font-style: normal;
+  font-weight: bold;
+  line-height: 1.3rem;
+  text-decoration: none;
+  font-size: 19px;
+  color: #6070ff;
```
Comment (binhussen) [inline style.css:44]: - [Optional] Kindly remove the default behavior of `a` tag, the logo inn the Figma hasn't been underlined so please stick to the Figma 👌👍

**Review by binhussen — APPROVED** [review]
Hi @Rishi-Mishra0704 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**